src/SDKs/CognitiveServices/dataPlane/AnomalyDetector/AnomalyDetector/Microsoft.Azure.CognitiveServices.AnomalyDetector.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.AnomalyDetector</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services AnomalyDetector APIs.</Description>
+    <Version>2.0</Version>
```

Comment:
You want directly publish a public release?
As it's first version, I personally think publishing a preview SDK first is a better idea?

Reply:
then make this to be 1.0.0.0